docs/source/_toctree.yml

```diff
@@ -52,6 +54,8 @@
       title: Mixins & serialization methods
     - local: package_reference/inference_api
       title: Inference API
+    - local: package_reference/hf_filesystem
+      title: Hugging Face Hub Filesystem
```

Comment:
I don't have a satisfying alternative but I'm not a fan of repeating huggingface hub in "Hugging Face Hub Filesystem" (since it's already huggingface hub docs). Any idea/opinion @stevhliu ?

Reply:
what about just naming it `hffs` in the doc? Would be the clearer way no?

Reply:
then just HfFileSystem?

Reply:
Yes, we can go with `HfFileSystem ` if @Wauplin @stevhliu agree

Reply:
> Yes, we can go with HfFileSystem 

Agree!